core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -247,6 +247,10 @@ private TableProperties() {
   public static final String SPLIT_SIZE = org.apache.iceberg.TableProperties.SPLIT_SIZE;
   public static final long SPLIT_SIZE_DEFAULT = 134217728; // 128 MB
 
+  public static final String SPLIT_TASK_BY_DELETE_RATIO = "read.split.delete-ratio";
```

Comment:
Currently, this feature is only effective within Trino. Is it appropriate to directly place the parameters on the table?